### DISCUSSION TITLE
Inpaint seamless tiling preperation node

### DISCUSSION THREAD

**Discussion by Ladypoly** [WASasquatch/was-node-suite-comfyui#181] [Ideas]
Can you integrate a node that simply prepared an image for tilling?
All it has to to do is split the image move it and add a mask over the middle seam. Then we can use in painting to create a perfect seamless texture.

![PreTiling](https://github.com/WASasquatch/was-node-suite-comfyui/assets/37795467/5d57f181-1b34-4224-9406-555e01399cba)

This is my current workaround which only works with 1024x1024 and only tiling on x. Everything else needs a lot of changing different nodes. 
![grafik](https://github.com/WASasquatch/was-node-suite-comfyui/assets/37795467/ff9c604b-1d44-4448-9709-d3f92bacb909)

**Comment by WASasquatch** [comment]
Wouldn't it be better to blend the tiles into each other via offsets and the img2img the result to fix any blending artifacts then hoping for inpainting to recreate the area perfectly?

**Reply by Ladypoly** [reply]
that should work too. I only tried to include all in one node that i don't have to create a mask.
In my tests it worked always pretty good to simply split and offset the image to only in paint the middle seam
